app/components/exercises.js

```diff
@@ -0,0 +1,11 @@
+import  React from 'react';
+
+export default class Exercises extends React.Component {
+
+  render() {
+    return (
+      <div>{this.props.exercises.map(exercise => <div>{exercise.name} - {exercise.reps}</div>)}</div>
```

Comment:
You may want to include a key property for element lists, see https://facebook.github.io/react/docs/lists-and-keys.html#keys